### ISSUE TITLE
Works perfectly in dev mode with an emulator but fails on Vercel and local build

### ISSUE THREAD

**Issue by yasaad** [ensite-in/next-firebase-auth-edge#24]
Hey Steve,

Really appreciate all the work you have put into this package so far. I got everything working well when running with the emulator. However, once I deploy to vercel or try running it from localhost without the emulator I see a failing call to `http://localhost:3000/api/login` is made. This call wasn't made when running on the local emulator. When I added some console logs in the middleware I see the following:

```
request.cookies RequestCookies {}
tokens null
Error: There is no user record corresponding to the provided identifier.
    at <unknown> (file:///Users/yasaad/Projects/upper-room-media/web-app/.next/server/middleware.js:378:15)
    at AuthRequestHandler.invokeRequestHandler (file:///Users/yasaad/Projects/upper-room-media/web-app/.next/server/middleware.js:438:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async getCustomIdAndRefreshTokens (file:///Users/yasaad/Projects/upper-room-media/web-app/.next/server/middleware.js:1406:24)
    at async setAuthCookies (file:///Users/yasaad/Projects/upper-room-media/web-app/.next/server/middleware.js:2953:39)
    at async adapter (file:///Users/yasaad/Projects/upper-room-media/web-app/.next/server/middleware.js:4263:20)
    at async (file:///Users/yasaad/Projects/upper-room-media/web-app/node_modules/.pnpm/next@13.1.6_pjwopsidmaokadturxaafygjp4/node_modules/next/dist/server/web/sandbox/sandbox.js:73:24)
    at async NextNodeServer.runMiddleware (file:///Users/yasaad/Projects/upper-room-media/web-app/node_modules/.pnpm/next@13.1.6_pjwopsidmaokadturxaafygjp4/node_modules/next/dist/server/next-server.js:1137:24)
    at async Object.fn (file:///Users/yasaad/Projects/upper-room-media/web-app/node_modules/.pnpm/next@13.1.6_pjwopsidmaokadturxaafygjp4/node_modules/next/dist/server/next-server.js:1216:38)
    at async Router.execute (file:///Users/yasaad/Projects/upper-room-media/web-app/node_modules/.pnpm/next@13.1.6_pjwopsidmaokadturxaafygjp4/node_modules/next/dist/server/router.js:253:36)
```

Initially I suspected that there was something wrong with my firebase settings but when I checked, the user is created and logged in. Here is a link to the branch that I am attempting to add the auth in: https://github.com/upperroommedia/web-app/tree/yasaad/next_13_routing

Please let me know if I can provide any other details to help!

**Comment by awinogrodzki** [comment]
Hey @yasaad,

Thanks for creating the issue. Some possible reasons for this come to my head, but probably I would need some more context on the setup and request itself.

Could you give me exact request and response data (including headers) for `/api/login` call? You can use Chrome DevTools Network tab to copy all the details.

If you're using Vercel, maybe you could share the url to your environment where error can be reproduced? It would really come in handy

By the way, the library has working example hosted [on Vercel](https://next-firebase-auth-edge-static-demo.vercel.app/). The source code can be found in [examples/next13-typescript-static-pages](https://github.com/ensite-in/next-firebase-auth-edge/tree/main/examples/next13-typescript-static-pages). Maybe it would be useful in some way.

**Comment by yasaad** [comment]
Hi @awinogrodzki,

After trying in all the wrong places, I found the issue. I first started by following the emulator example. When I moved to the static pages example I forgot to remove the `FIREBASE_AUTH_EMULATOR_HOST` from the `.env` file (which I also uploaded to Vercel). I'm not really sure how the rest of the app was still working with the actual database but removing this form the .env file and Vercel environment variables did the trick!

Thanks for the quick response!
